### ISSUE TITLE
All cutscenes in Super Mario RPG are slow motion and drop frames

### ISSUE THREAD

**Issue by Chapadin** [Ryujinx/Ryujinx#5965]
### Description of the issue

For some reason, all the cutscenes in the game are in slow motion (or when the castle is in the area of ​​Bowser's castle) and start to require a high CPU usage rate, while the rest of the game can run at an acceptable frame rate. Here are the latest log files
![ryujinx_capture_2023-11-19_22-08-05](https://github.com/Ryujinx/Ryujinx/assets/147850373/59e450cc-7911-40fa-a220-050100a5b846)

### Reproduction steps

All the game's cutscenes are slow and when I'm in the Bowser's castle area, it generates high CPU usage. In the meantime, I can play the rest of the game at an acceptable framerate (outside of the cutscenes).

### Log file

[Ryujinx_1.1.1092_2023-11-20_09-06-34.log](https://github.com/Ryujinx/Ryujinx/files/13413784/Ryujinx_1.1.1092_2023-11-20_09-06-34.log)
[Ryujinx_1.1.1092_2023-11-20_09-06-36.log](https://github.com/Ryujinx/Ryujinx/files/13413786/Ryujinx_1.1.1092_2023-11-20_09-06-36.log)

### OS

Windows 11

### Ryujinx version

1.1.1092

### Game version

1.0.0

### CPU

Intel Core i3 1115G4

### GPU

Intel UHD Graphics for 11th Gen Intel Processors

### RAM

8GBs

**Comment by RupeeClock** [comment]
Many cutscenes are pre-rendered video that run at 60 FPS and utilise nvdec for decoding. Video stream is run separate to audio, which can cause desync if video playback slows or stalls.
It appears that hardware without fast enough video decoding or insufficient video memory experience slowdown, such as running Intel integrated graphics or low-end graphics cards.
See comments on the [Game Compatibility entry](https://github.com/Ryujinx/Ryujinx-Games-List/issues/4811#issuecomment-1817916988).

**Comment by gdkchan** [comment]
Ryujinx does not have hardware video decoding, so your GPU does not matter, it depends entirely on the CPU in this case.

**Comment by RupeeClock** [comment]
I see, I'd learned the wrong thing from the Game Compatibility entry then, my bad.
Would this mean nvdec is referring to the emulated Switch system's video decoding capabilities?
When investigating the issue recently, I did observe that when a video cutscene is loaded by SMRPG, that GPU VRAM usage would increase by around 1GB, which lead me to believe it was handling video decoding.

**Comment by gdkchan** [comment]
Yes, the support has nvdec, and so the emulator has to emulate it too. It doesn't mean that it is emulated using the host hardware video decoding capabilities.

**Comment by martinsaenz96** [comment]
I've been having this issue too, and I'm playing the game through Steam Deck. I thought it was the emulator that's causing the issue, so I stopped playing for a while. Now testing it the issue still persist, even after installing the latest firmware.
